contracts/consensus/Consensus.sol

```diff
@@ -110,15 +123,18 @@ contract Consensus is MasterCopyNonUpgradable, CoreStatusEnum, ConsensusI {
     mapping(address => address) public committees;
 
 // NOTE: consider either storing a linked list; or getting rid of it
```

Comment:
you can remove this note pls; now the mapping for `anchors` is more sensible and we'll keep it as this